public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,84 @@
+# dvc.api.get_url()
+
+Return the URL
+([string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)
+type)) to the storage location of a <abbr>data artifact</abbr> tracked by DVC.
+
+## Signature
+
+```py
+get_url(path, repo=None, rev=None, remote=None)
+```
+
+## Description
+
+Returns the full URL to the physical location (in a
+[DVC remote](/doc/command-reference/remote)) where a target data artifact
+specified by its `path` in a `repo` (<abbr>DVC project</abbr>) is stored. The
+URL is formed by evaluating the corresponding
+[DVC-file](/doc/user-guide/dvc-file-format) (see [Examples](#examples) below) as
+well as the project's
+[default remote](https://dvc.org/doc/command-reference/remote/default).
+
+⚠️ This function does not check for the actual existence of the file or
+directory in the remote storage.
+
+💡 Having the resource's URL, it should be possible to download it directly with
+an appropriate tool such as
+[`urlretrieve`](https://docs.python.org/3/library/urllib.request.html#urllib.request.urlretrieve)
+or `boto3`
+[download_fileobj](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Object.download_fileobj).
+
+Note that if the target is a directory, the URL will end in `.dir`, as DVC
+stores a special JSON file with `.dir` extension that contains the mapping of
+files in the directory (as a JSON array), along with their checksums. Refer to
+[Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
+and `dvc add` to learn more about how DVC handles data directories.
+
+> For possible location protocols, refer to the
+> [supported remote types](https://dvc.org/doc/command-reference/remote/add#supported-storage-types)
+
+## Parameters
+
+- **`path`** - specifies the location of the target artifact within the source
```

Comment:
`location of the data file or directory relative to ...`

Reply:
Hmmm. I don't know. I've changed "artifact" for "file or directory" in the description of this fn but I'm not sure directories are supported in `open()` or especially `read()`. I'll double check first ⏳

Reply:
Nope, 'open()' and thus 'read()' don't support directories so lets keep "target artifact" here? OR maybe just "target data" since the "artifact" glossary entry also reads "file or directory"... Updated, please check!